cockpit-frontend/src/components/OverviewSprintTabs/OverviewSprintTabs.jsx

```diff
@@ -118,17 +118,17 @@ export default function OverviewSprintTabs(props) {
           <CardContent>
             <Grid container alignItems="center">
               <Grid item xs={12}>
-                <Typography variant="h6" color="textSecondary">
-                  MAIN IMPEDIMENTS
+                <Typography variant="subtitle1" color="textSecondary">
```

Comment:
the text "main impediments" would look better in Bold style, could you change it plz

Reply:
done